docs/learn-identity.md

```diff
@@ -4,27 +4,63 @@ title: Identity
 sidebar_label: Identity
 ---
 
-Polkadot provides a naming system that allows participants to ask for verification of off-chain information from registrars. Users can ask for verification of multiple attributes (e.g. name, website, image). 
+Polkadot provides a naming system that allows participants to add personal information to their on-chain account and subsequently ask for verification of this information by [registrars](#registrars).
+
+## Setting an Identity
+
+Users can register some default fields like legal name, display name, website, Twitter handle, Riot handle, etc. along with extra, custom fields for which they would like attestations (see [Judgements](#judgements)). Users must reserve funds in a bond to store their information on chain - 10 per identity, and 2.5 KSM per each field beyond the legal name. These funds are locked and not spent - they are returned when the identity is cleared. Each field can store up to 32 bytes of information, so the data must be less than that. When inputting the data manually through the [Extrinsics UI](https://polkadot.js.org/apps/#/extrinsics), a [UTF8 to bytes](https://onlineutf8tools.com/convert-utf8-to-bytes) converter can help.
+
+The easiest way to add the built-in fields is to click the gear icon next to one's account and select "Set on-chain identity".
+
+![Gear icon provides the option to set identity](/img/identity/01.jpg)
+
+A popup will appear, offering the default fields.
+
+![Identity field setup popup](/img/identity/02.jpg)
+
+To add custom fields beyond the default ones, use the Extrinsics UI to submit a raw transaction by first clicking "Add Item" and adding any field you desire. The example below adds a field `steam` which is a user's [Steam](https://store.steampowered.com) username. The first value is the field name in bytes ("steam") and the second is the account name in bytes ("theswader"). The display name also has to be provided, otherwise the Identity pallet would consider it wiped if we submitted it with the "None" option still selected. That is to say, every time you make a change to your identity values, you need to re-submit the entire set of fields: the write operation is always "overwrite", never "append".
+
+![Setting a custom field](/img/identity/03.jpg)
+
+Note that custom fields are not shown in the UI by default:
+
+![Only built-in fields are shown](/img/identity/04.jpg)
+
+The rendering of such custom values is, ultimately, up to the UI/dapp makers. In the case of PolkadotJS, the team prefers to only show official fields for now. If you want to check that the values are still stored, use the [Chain State UI](https://polkadot.js.org/apps/#/chainstate) to query the active account's identity info:
+
+![Raw values of custom fileds are available on-chain](/img/identity/05.jpg)
+
+It is up to your own UI or dapp to then do with this data as it pleases. The data will remain available for querying via RPC calls, so you don't have to rely on the PolkadotJS UI.
```

Comment:
```suggestion
It is up to your own UI or dapp to then do with this data as it pleases. The data will remain available for querying via the Polkadot API, so you don't have to rely on the PolkadotJS UI.
```

I know that the Polkadot API wraps RPC calls, but I think this makes more sense since they'd probably call the API.